app/config.yml

```diff
@@ -3,11 +3,17 @@
 workspace_path: /tmp
 # name or path of logger file
 logger_file_name: restylinchpin.log
-# path to tinydb source file
+#path to users tinydb source file
+users_db_path: users.json
```

Comment:
@mansikulkarni96 
Do we need to have another json file, 
can use the same db.json and create another record for users ?? 
I don't see any advantages on using another db for the same application